src/ServiceControl.Monitoring.Tests/PerformanceTests.cs

```diff
@@ -244,5 +259,55 @@ static void Report(string name, LongHistogram histogram, TimeSpan? maximumMean)
                 Assert.LessOrEqual(actualMean, max, $"The actual mean for {name} was '{actualMean}' and was bigger than maximum allowed mean '{max}'.");
             }
         }
+
+        interface IStoreMetric
```

Comment:
This could be replaced by a lambda `Action<RawMessage.Entry[], EndpointInstanceId instanceId> store` in `StartReporter` with no need of creating wrappers and the interface itself.